bindings/cs/rl.net.cli/EntryPoints.cs

```diff
@@ -41,20 +41,27 @@ private static LiveModel CreateLiveModelOrExit(string clientJsonPath)
                 WriteStatusAndExit(apiStatus);
             }
 
-            LiveModel liveModel = new LiveModel(config);
+            LiveModel liveModel = new LiveModel(config);
+
+            liveModel.BackgroundError += LiveModel_BackgroundError;
+            liveModel.TraceLoggerEvent += LiveModel_TraceLogEvent;
+
             if (!liveModel.TryInit(apiStatus))
             {
                 WriteStatusAndExit(apiStatus);
             }
 
-            liveModel.BackgroundError += LiveModel_BackgroundError;
-
             return liveModel;
         }
 
         private static void LiveModel_BackgroundError(object sender, ApiStatus e)
         {
             Console.Error.WriteLine(e.ErrorMessage);
+        }
+
+        private static void LiveModel_TraceLogEvent(object sender, TraceLogEventArgs e)
+        {
+            Console.WriteLine($"LogLevel: {e.LogLevel} LogMessage: {e.Message}");
         }
```

Comment:
Since we have access to the LogLevel I would suggest using "Console.Error" when appropriate.

Reply:
Fixed.